src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/mobile/PostsCreatorMobile.java

```diff
@@ -23,6 +23,10 @@
   private WebElement signInButtonInSignInDialog;
 
   @Getter
+  @FindBy (css = ".discussion-standalone-editor")
```

Comment:
`className` strategy possible - do we have any consistent rules (or practices) when to use which?